### PR TITLE
Support HWB colors in SassColor

### PR DESCRIPTION
**Pull request by nex3**
See sass/sass#2835

### CODE REVIEW

File: lib/src/value/color.dart [hunk 1]
```diff
@@ -55,6 +55,20 @@ class SassColor extends Value implements ext.SassColor {
 
   num _lightness;
 
+  num get whiteness {
+    // Because HWB is (currently) used much less frequently than HSL or RGB, we
+    // don't cache its values because we expect the memory overhead of doing so
+    // to outweight the cost of recalculating it on access.
```
Comment (Awjin) [inline lib/src/value/color.dart:61]: nit:

```suggestion
    // to outweigh the cost of recalculating it on access.
```

File: lib/src/value/color.dart [hunk 2]
```diff
@@ -55,6 +55,20 @@ class SassColor extends Value implements ext.SassColor {
...
+    return math.min(math.min(red, green), blue) / 255 * 100;
+  }
+
+  num get blackness {
+    // Because HWB is (currently) used much less frequently than HSL or RGB, we
+    // don't cache its values because we expect the memory overhead of doing so
+    // to outweight the cost of recalculating it on access.
```
Comment (Awjin) [inline lib/src/value/color.dart:68]: nit:

```suggestion
    // to outweigh the cost of recalculating it on access.
```

File: lib/src/value/color.dart [hunk 3]
```diff
@@ -81,6 +95,31 @@ class SassColor extends Value implements ext.SassColor {
...
+      scaledBlackness /= sum;
+    }
+
+    var factor = 1 - scaledWhiteness - scaledBlackness;
+    int toRgb(num hue) {
+      var channel = _hueToRgb(0, 1, hue) * factor + scaledWhiteness;
+      return fuzzyRound(channel * 255);
+    }
+
+    return SassColor.rgb(
```
Comment (Awjin) [inline lib/src/value/color.dart:116]: It could be worth documenting why this returns an rgb color.

**Review by Awjin — APPROVED** [review]
Needs a pubspec bump, otherwise the code LGTM.

File: lib/src/value/color.dart (hunk 3 above)
Comment (nex3) [reply lib/src/value/color.dart:120]: Done.

File: lib/src/value/color.dart (hunk 1 above)
Comment (nex3) [reply lib/src/value/color.dart]: Done.

File: lib/src/value/color.dart (hunk 2 above)
Comment (nex3) [reply lib/src/value/color.dart]: Done.

**Review by nex3 — COMMENTED** [review]
> Needs a pubspec bump, otherwise the code LGTM.

Done.

**Pull request merged**